examples/get-started/hook/map.jsx

```diff
@@ -1,14 +1,38 @@
 import * as React from 'react';
-import Map from 'react-map-gl';
+import Map, {useMap} from 'react-map-gl';
 
 import 'mapbox-gl/dist/mapbox-gl.css';
+import Controls2 from './controls2';
 
 const MAPBOX_TOKEN = ''; // Set your mapbox token here
 
 export default function MapView() {
+  /**
+   * ## This is how `useMap` works:
+   * ```
+   * const demo = useMap();
+   * console.log('MapView', {demo});
+   * ```
+   * ### First render:
+   * ```
+   * {
+   *     "demo": {
+   *         "current": undefined
+   *     }
+   * }
+   * Second render:
+   * {
+   *     "demo": {
+   *         "current": undefined,
+   *         "mymap": {...} // See https://visgl.github.io/react-map-gl/docs/api-reference/types#mapref
+   *     }
+   * }
+   * ```
```

Comment:
```suggestion
   * ```
   * const maps = useMap();
   * console.log('MapView useMap()', maps);
   * ```
   * ### First render:
   * ```
   * {
   *     "current": undefined
   * }
   * Second render:
   * {
   *     "current": undefined,
   *     "mymap": {...} // See https://visgl.github.io/react-map-gl/docs/api-reference/types#mapref
   * }
   * ```
```